### DISCUSSION TITLE
Autoscoping of CSS

### DISCUSSION THREAD

**Discussion by y2kbugger** [jpsca/jinjax#43] [General]
Is autoscoping of CSS on the roadmap for future or is it a design choice to avoid that?

Also what about auto-adding assets via naming convention e.g. `Card.jinja` searches for `Card.css` or `Card.jinja.css`?

**Comment by jpsca** [comment]
1. Autoscoping is not completely... out of the scope, but not in the near roadmap either because it would require parsing the CSS and replace the new names in the Jinja HTML and I suspect it would be tricky to debug all the corner cases.
2. Auto-adding assets sounds like a great idea!
